.github/scripts/createLicenseDisclaimerPr.ts

```diff
@@ -21,7 +21,7 @@ exec('git config user.name "valora-bot"')
 exec('git commit -m "Update licenses and disclaimer"')
 exec(`git push --set-upstream origin ${branchName}`)
 
-echo('Open licenses and disclaimer PR')
+echo('chore: open licenses and disclaimer PR')
```

Comment:
I think this change was meant to be a few lines down 😄

Reply:
lollllll oops, brain fart. updated!